fortitude/src/cli.rs

```diff
@@ -97,16 +97,45 @@ pub struct CheckArgs {
     #[arg(long, value_delimiter = ',', default_values = FORTRAN_EXTS)]
     pub file_extensions: Option<Vec<String>>,
 
+    /// Apply fixes to resolve lint violations.
+    /// Use `--no-fix` to disable or `--unsafe-fixes` to include unsafe fixes.
+    #[arg(long, overrides_with("no_fix"), action = clap::ArgAction::SetTrue)]
+    pub fix: Option<bool>,
+    #[clap(long, overrides_with("fix"), hide = true, action = SetTrue)]
+    pub no_fix: Option<bool>,
+    /// Include fixes that may not retain the original intent of the code.
+    /// Use `--no-unsafe-fixes` to disable.
+    #[arg(long, overrides_with("no_unsafe_fixes"), action = SetTrue)]
+    pub unsafe_fixes: Option<bool>,
+    #[arg(long, overrides_with("unsafe_fixes"), hide = true, action = SetTrue)]
+    pub no_unsafe_fixes: Option<bool>,
+    /// Show an enumeration of all fixed lint violations.
+    /// Use `--no-show-fixes` to disable.
+    #[arg(long, overrides_with("no_show_fixes"), action = SetTrue)]
+    pub show_fixes: Option<bool>,
+    #[clap(long, overrides_with("show_fixes"), hide = true, action = SetTrue)]
+    pub no_show_fixes: Option<bool>,
+    /// Apply fixes to resolve lint violations, but don't report on, or exit non-zero for, leftover violations. Implies `--fix`.
+    /// Use `--no-fix-only` to disable or `--unsafe-fixes` to include unsafe fixes.
+    #[arg(long, overrides_with("no_fix_only"), action = SetTrue)]
+    pub fix_only: Option<bool>,
+    #[clap(long, overrides_with("fix_only"), hide = true, action = SetTrue)]
+    pub no_fix_only: Option<bool>,
+    /// Avoid writing any fixed files back; instead, output a diff for each changed file to stdout, and exit 0 if there are no diffs.
+    /// Implies `--fix-only`.
+    #[arg(long, conflicts_with = "show_fixes", action = SetTrue)]
+    pub diff: Option<bool>,
```

Comment:
This isn't actually implemented yet, maybe remove for now?

Reply:
Sure, there's already quite a lot of new features to get to grips with, so removing it will reduce the complexity a little.